app/controllers/redirect/redirect_controller.rb

```diff
@@ -10,14 +10,35 @@ def location_params
       params.permit(*PERMITTED_LOCATION_PARAMS).to_h.symbolize_keys
     end
 
+    def partner_query_params
+      @partner_query_params ||= begin
+        {
+          partner: current_sp&.agency&.name,
+          partner_div: current_sp&.integration&.name,
```

Comment:
where did the name `partner_div` come from? is `div` an abbreviation?

Reply:
These names were left intentionally vague as the user could potentially notice them (they will be removed from the URL once read in identity-site).

I expect some names will continue to change. Open to suggestions.

Reply:
I feel like the fact that a user could notice them means we should make it clearer what they are, like `agency_name` and `app_name` or something

Reply:
The value partner cleanly maps to `agency.name`. The definitive name for `integration` is less clear to me, so I left it open ended at the moment.

Reply:
That may have been a little unclear. The values for the variable `agency.name` are clean agency names. Elsewhere, such as the contact form itself, some of the naming conventions get mixed (not all agencies there are agencies). The definitive name for the values found in `integration` I've left open ended.